src/coreclr/gc/gc.cpp

```diff
@@ -51270,28 +51270,28 @@ size_t gc_heap::get_gen0_min_size()
 #ifdef SERVER_GC
         // performance data seems to indicate halving the size results
         // in optimal perf.  Ask for adjusted gen0 size.
-        gen0size = max(GCToOSInterface::GetCacheSizePerLogicalCpu(FALSE),(256*1024));
+        gen0size = max(GCToOSInterface::GetCacheSizePerLogicalCpu(FALSE), (size_t)(256*1024u));
```

Comment:
The `u` should no longer be needed if we are doing a cast.

Several places in this file do that. I would do a cast or use the `u`, but not both.